rst_files/finite_markov.rst

```diff
@@ -1384,7 +1384,7 @@ compare it to the stationary probability.
             X = simulate_indices(mc, N; init = x0)
 
             # == Compute fraction of time spent unemployed, for each n == #
-            X_bar = cumsum(X.==1) ./ 1:N
+            X_bar = cumsum(X.==1) ./ collect(1:N)
```

Comment:
I think this is a precedence thing rather than broadcasting with ranges.  Try `X_bar = cumsum(X.==1) ./ (1:N)` to see if adding brackets helps.

Reply:
Can confirm this is the preferred solution. You can make a small edit to change `collect` to a `()`.

Reply:
Stick with the brackets

Reply:
OK, will change this to `[1:N]`.

Reply:
Done.